content/en/legacy/flux/references/garbagecollection.md

```diff
@@ -0,0 +1,48 @@
+---
+type: legacy
+title: Garbage collection
+---
+
+Part of syncing a cluster with a git repository is getting rid of
+resources in the cluster that have been removed in the repository. You
+can tell `fluxd` to do this "garbage collection" using the command-line
+flag `--sync-garbage-collection`. It's important to know how it
+operates, and appreciate its limitations, before enabling it.
+
+## How garbage collection works
+
+When garbage collection is enabled, syncing is done in two phases:
+
+ 1. Apply all the manifests in the git repo (as delimited by the
+    branch and path arguments), and give each resource a label marking
+    it as having been synced from this source.
+
+ 2. Ask the cluster for all the resources marked as being from this
+    source, and delete those that were not applied in step 1.
+
+In the above, "source" refers to the particular combination of git
+repo URL, branch, and paths that this `fluxd` has been configured to
+use, which is taken as identifying the resources under _this_
+`fluxd`'s control.
+
+We need to be careful about identifying these accurately, since
+getting it wrong could mean _not_ deleting resources that should be
+deleted; or (much worse), deleting resources that are under another
+`fluxd`'s control.
+
+The definition of "source" affects how garbage collection behaves when
+you reconfigure `fluxd`. It is intended to be conservative: it ensures
+that `fluxd` will not delete resources that it did not create.
+
+## Limitations of this approach
+
+In general, if you change an element of the source (the git repo URL,
+branch, and paths), there is a possiblility that resources no longer
+present in the new source will be missed (i.e., not deleted) by
+garbage collection, and you will need to delete them by hand.
+
```

Comment:
Similarly unreadable table

New:
<img width="1324" alt="Screen Shot 2021-06-02 at 3 30 40 PM" src="https://user-images.githubusercontent.com/3286998/120541271-acc5ae80-c3b7-11eb-8852-ba6d79d6c03e.png">

Old:
<img width="1077" alt="Screen Shot 2021-06-02 at 3 32 23 PM" src="https://user-images.githubusercontent.com/3286998/120541333-c0711500-c3b7-11eb-9c26-b16ca460053f.png">

I think this should be fixed by the same change that fixes any other table issue. I would suggest an alternate color or border-based callout look for the table header (light gray background like the old version would be fine IMHO) and a bit more whitespace in the form of padding or margin between the columns.

Reply:
maybe it's just a styling bit that's missing from the other docs...?